### ISSUE TITLE
Notification and Chat bubble in the navbar

### ISSUE THREAD

**Issue by MathiasOliveira** [jeroennoten/Laravel-AdminLTE#713]
| Question                | Answer
| ----------------------- | -----------------------
| Issue or Enhancement    | Question
| Laravel Version         | v8.4.0
| Project Version         | v3.4.4

Sorry for posting this question, but I couldn't find anything in the docs.

Is it possible to include the notification bell and chat bubble on the navbar in the left side of the username like in the original adminlte3?
![Screenshot_2](https://user-images.githubusercontent.com/67007152/94015269-04207300-fd83-11ea-80f3-58e617353d08.png)

**Comment by Shidersz** [comment]
Hi @MathiasOliveira , it is not nicely supported right now, you can use something like this:

```php
[
    'text' => '',
    'url' => '#',
    'label'   => 10,
    'label_color' => 'danger',
    'topnav_right' => true,
    'icon' => 'fas fa-bell',
],
```

But the badge will show at the right (not at the upper-right). I'm going to work on a way to include these type of menu items on the `navbar`, do you expect to use it as a dropdown menu?

**Comment by MathiasOliveira** [comment]
Yes, a dropdown with a list of notifications in the popover, like this:
![Screenshot_4](https://user-images.githubusercontent.com/67007152/94040772-b9612400-fd9f-11ea-9e11-ccdf79968894.png)

**Comment by Shidersz** [comment]
Ok, the support for this new type of menu items will require some work. @lacodimizer Please, schedule this as new feature. I will get on it on my free time.

**Comment by x7ryan** [comment]
Just checking but I don't suppose this has been done yet? One related enhancement would be to go so far as to integrate directly with Laravel's database notifications system, including marking individual or all notifications as read or unread. So all I would have to do is set a config value to enable it, set the icon, and choose any other relevant settings.

I guess for now I could implement manually as you showed above but I just thought I'd check on the status of this one since its been a few months.

**Comment by Shidersz** [comment]
@x7ryan Unfortunately, I had no time to work on this feature, since I was (and I'm) putting all my effort on implementing a set of blade-x components for the package and the integration of **AdminLTe v3.1.0**. However, I have analyzed this situation in the past and the dropdown content can be too custom that will be hard to satisfy all the needs. You can see the next two examples:

![Screenshot_2021-04-12 AdminLTE 3 Dashboard(1)](https://user-images.githubusercontent.com/63609705/114403461-3062dc00-9b7b-11eb-8f02-86cc718f5921.png)

![Screenshot_2021-04-12 AdminLTE 3 Dashboard](https://user-images.githubusercontent.com/63609705/114403462-30fb7280-9b7b-11eb-866e-36a3f5632699.png)

My first idea was just to provide a way to define the icon with the label and an empty dropdown yielding a section, so you can fill the section later. But that is so basic, that I prefer to wait until I have more time to think for a better solution. So, for the moment I have no plan for this feature, but I will review it on the future.

**Comment by x7ryan** [comment]
My thought was a config to choose an icon, badge color, an option to enable a built in list of Laravel Database Notifications for a user or the ability to specify a custom handler to build a custom drop down for more advanced needs like the messaging example in your screenshot.

But honestly even just a config to set icon, badge color, and specify a custom handler to build the drop down or configure it as a link would be sufficient for now.

**Comment by Shidersz** [comment]
@x7ryan @MathiasOliveira I have created the draft PR #888 to work on the support for a new notification link. The first one will redirect you to a configurable **url** when clicking on it. However, I'm planing to create a new one that will support a dropdown instead of the redirect `url`. Details can be found on the PR, any feedback is welcome...
